src/samples/msal-web-sample/src/main/resources/application.properties

```diff
@@ -1,5 +1,15 @@
 aad.authority=https://login.microsoftonline.com/common/
 aad.clientId=
 aad.secretKey=
-aad.redirectUri=http://localhost:8080/msal4jsample/secure/aad
-aad.oboApi=<OboApi>/access_as_user
\ No newline at end of file
+aad.redirectUri=https://localhost:8081/msal4jsample/secure/aad
+aad.oboApi=<OboAi>/access_as_user
+
+server.servlet.session.cookie.secure=true
```

Comment:
Is secure cookie necessary as part of the SameSite changes? 

I realize that this is a necessary practice in a production app, but it had been omitted in this samples previously because it further complicates getting the sample running. For the Azure Samples and the portal quickstart, these changes will have to be explained, including instructions on how to set this up. IMO a section in the README and in a comment in the code could be sufficient.

Reply:
In new versions of Chrome cookies with same site none will not be included in cs request if they not secure